examples/chart/teleport-cluster/templates/proxy/deployment.yaml

```diff
@@ -109,6 +109,10 @@ spec:
             - wait
             - no-resolve
             - '{{ include "teleport-cluster.auth.previousVersionServiceName" . }}.{{ .Release.Namespace }}.svc.cluster.local'
+{{- if $proxy.resources }}
+          resources:
+  {{- toYaml $proxy.resources | nindent 12 }}
```

Comment:
We should add a specific resources to the wait container otherwise this will request twice the number of cpus and might not fit the customer node

Reply:
@tigrato @hugoShaka Do you think maybe I just hardcode this to 0.1cpu and 256Mi RAM or something? I'd rather not expose new values for this when really it's just a bug fix, and it's a tiny container that does nothing except wait.

Reply:
Should be better.
If `$proxy.resources ` is set, hardcode it to 0.1cpu + 256Mb